app/build.gradle

```diff
@@ -31,6 +33,11 @@ android {
         testInstrumentationRunner = "android.support.test.runner.AndroidJUnitRunner"
     }
 
+    sourceSets {
+        // May no longer be needed in future versions of Android Studio or the Kotlin Gradle plugin?
+        main.java.srcDirs += 'src/main/kotlin'
```

Comment:
Make `src/main/kotlin/` a valid source directory. We place our Kotlin (.kt) files in the `kotlin/` sourceSet instead of the `java/` sourceSet.